TestAssets/ProjectModelServer/MSBuildReferencesProjects/BasicCase01/ClassLibrary1/Class1.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
This should be more expressive than BasicCase01. What does this project represent?